mm2src/coins/eth/web3_transport/websocket_transport.rs

```diff
@@ -87,10 +87,9 @@ impl WebsocketTransport {
             event_handlers,
             request_id: Arc::new(AtomicUsize::new(1)),
             controller_channel: ControllerChannel {
-                tx: Arc::new(AsyncMutex::new(req_tx)),
+                tx: req_tx,
                 rx: Arc::new(AsyncMutex::new(req_rx)),
-            }
-            .into(),
+            },
```

Comment:
```rust
rx: AsyncMutex::new(req_rx)
```

Reply:
that arc was indeed the unnecessary one, thanks :)
[8ae149b](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2291/commits/8ae149bc194c4bab48a146a8b373e9a74e52306e)